src/WorkflowCore.DSL/Services/DefinitionLoader.cs

```diff
@@ -68,8 +68,22 @@ private WorkflowStepCollection ConvertSteps(ICollection<StepSourceV1> source, Ty
                 var nextStep = stack.Pop();
 
                 var stepType = FindType(nextStep.StepType);
-                var containerType = typeof(WorkflowStep<>).MakeGenericType(stepType);
-                var targetStep = (containerType.GetConstructor(new Type[] { }).Invoke(null) as WorkflowStep);
+
+                WorkflowStep targetStep;
+
+                Type containerType;
+                if (stepType.IsSubclassOf(typeof(StepBody)))
```

Comment:
Hello, 

Not sure you will handle all step with typeof(StepBody). (missing StepBodyAsync)
Would it not be better to use stepType.GetInterfaces().Contains(typeof(IStepBody)) ?

Reply:
Yes,you are right.I missed `StepBodyAsync`